src/Components/Facility/BedManagement.tsx

```diff
@@ -57,6 +60,34 @@ const BedRow = (props: BedRowProps) => {
   const [descField, setDescField] = useState(description);
   const [bedTypeField, setBedTypeField] = useState(bedType);
   const [isLoading, setIsLoading] = useState(false);
+  const [bedData, setBedData] = useState<{
+    show: boolean;
+    id: string;
+    name: string;
+  }>({ show: false, id: "", name: "" });
+
+  const handleDelete = (name: string, id: string) => {
+    setBedData({
+      show: true,
+      id,
+      name,
+    });
+  };
+
+  const handleDeleteConfirm = async () => {
+    const res = await dispatchAction(deleteFacilityBed(id));
+    if (res && res.status == 204) {
+      Notification.Success({
+        msg: "Bed deleted successfully",
+      });
+    }
+    setBedData({ show: false, id: "", name: "" });
+    window.location.reload();
```

Comment:
Don't reload the page (reloading the page is slow ). Either remove row from a table or refetch `bed data` after successful deletion

Reply:
Refetch would be the right approach

Reply:
Replaced it with `triggerRerender();` . (`handleSave` uses the same for a re render)